app/assets/javascripts/discourse/app/components/reviewable-item.js

```diff
@@ -299,22 +299,15 @@ export default Component.extend({
       const requireRejectReason = performableAction.get(
         "require_reject_reason"
       );
-      const actionModalClass =
-        actionModalClassMap[performableAction.server_action];
+      const actionModalClass = requireRejectReason
+        ? RejectReasonReviewableModal
+        : actionModalClassMap[performableAction.server_action];
```

Comment:
I wonder if `performableAction.server_action` is `require_reject_reason`? If so maybe we can just add `RejectReasonReviewableModal` to `defaultActionModalClassMap`?

Reply:
If not what you have is fine :+1: